package/Dockerfile.arm

```diff
@@ -1,4 +1,4 @@
-FROM alpine@sha256:6d94d2806ccbe3b6036b2cd55c61260f474692b6987c21386e32ef54bc40200a
+FROM alpine:3.12
```

Comment:
if we're going to use alpine:3.12 for everything you could probably just drop the arm-specific dockerfile entirely.

Reply:
good point